hypertension/.vscode/settings.json

```diff
@@ -4,5 +4,5 @@
         "!Ref",
         "!GetAtt"
     ],
-    "python.pythonPath": "/Users/nhillard/.pyenv/versions/3.8.6/bin/python3.8"
+    "python.pythonPath": "/Users/julialallen/.pyenv/versions/3.8.6/bin/python"
```

Comment:
I didn't mean to commit this change, although I lack motivation to undo it also.

Reply:
Should the `.vscode/settings.json` file be in the repo at all?

Reply:
Maybe not - the tweaks for debugging and intellisense are useful if you're using vscode, so I kinda like the idea of sharing those. Maybe in a development doc instead?

Reply:
How about this instead? https://github.com/department-of-veterans-affairs/Virtual-Regional-Office/pull/15/commits/e63356c311077f5321e5da1c367bfac23d46d102

Reply:
Looks like this `python.pythonPath` line didn't make it into `docs/vs-code.md`.

Do you know what this line does? Should it also be in `docs/vs-code.md`.

I don't know what it does, but I'm wondering if it is related to the info I learned from Nat Hillard and wrote here:
https://github.com/department-of-veterans-affairs/Virtual-Regional-Office/blob/develop/hypertension/docs/design-notes.md#python-environment